samples/core/parallelism_sub_dag/parallelism_sub_dag.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import kfp.dsl as dsl
+import kfp
+
+@kfp.components.create_component_from_func
+def print_op(s: str):
+    print(s)
+
+@dsl.pipeline(name='my-pipeline')
+def pipeline():
+    loop_args = [{'A_a': 1, 'B_b': 2}, {'A_a': 10, 'B_b': 20}]
+    with dsl.SubGraph(parallelism=2):
+        with dsl.ParallelFor(loop_args) as item:
+            print_op(item)
```

Comment:
nit: Is this sample intended to demonstrate how the `parallelism` impact the number of tasks that can be simultaneously executed? If so I'll suggest add a sleep in `print_op`. Otherwise it still looks like all 6 tasks finish at the same time because they are very lightweight.

Reply:
Good point, will add.